REUSE.toml

```diff
@@ -0,0 +1,11 @@
+version = 1
+SPDX-PackageName = "elektra"
+SPDX-PackageSupplier = "ospo@sap.com"
+SPDX-PackageDownloadLocation = "https://github.com/sapcc/elektra"
+SPDX-PackageComment = "The code in this project may include calls to APIs (\"API Calls\") of\n SAP or third-party products or services developed outside of this project\n (\"External Products\").\n \"APIs\" means application programming interfaces, as well as their respective\n specifications and implementing code that allows software to communicate with\n other software.\n API Calls to External Products are not licensed under the open source license\n that governs this project. The use of such API Calls and related External\n Products are subject to applicable additional agreements with the relevant\n provider of the External Products. In no event shall the open source license\n that governs this project grant any rights in or to any External Products,or\n alter, expand or supersede any terms of the applicable additional agreements.\n If you have a valid license agreement with SAP for the use of a particular SAP\n External Product, then you may make use of any API Calls included in this\n project's code for that SAP External Product, subject to the terms of such\n license agreement. If you do not have a valid license agreement for the use of\n a particular SAP External Product, then you may only make use of any API Calls\n in this project for that SAP External Product for your internal, non-productive\n and non-commercial test and evaluation of such API Calls. Nothing herein grants\n you any rights to use or access any SAP External Product, or provide any third\n parties the right to use of access any SAP External Product, through API Calls."
+
+[[annotations]]
+path = "**"
```

Comment:
Is this a problem when the repo contains files licensed under something different? https://github.com/sapcc/elektra/blob/1acdf0d1f7c0b97bef71ac6305f7c59f0c6a151a/app/javascript/core/jquery.bbq.js#L6

Reply:
Actually not a problem, but an indicator that we should extend the declaration in REUSE.toml. Can you point me to the copied files/sections? I can take care of the rest.

Reply:
I was just curious for our projects and can't help you with that 😅